pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/ModularLoadManagerStrategy.java

```diff
@@ -55,11 +58,17 @@ Optional<String> selectBroker(Set<String> candidates, BundleData bundleToAssign,
      */
     static ModularLoadManagerStrategy create(final ServiceConfiguration conf) {
         try {
-            // Only one strategy at the moment.
-            return new LeastLongTermMessageRate(conf);
+            Class<?> loadManagerStrategyClass = Class.forName(conf.getLoadBalancerLoadManagerStrategy());
+            Object loadManagerStrategyInstance = loadManagerStrategyClass.newInstance();
+            if (loadManagerStrategyInstance instanceof ModularLoadManagerStrategy) {
+                return (ModularLoadManagerStrategy) loadManagerStrategyInstance;
+            } else {
+                LOG.error("create load manager strategy failed. using LeastLongTermMessageRate instead.");
```

Comment:
please throw a error, otherwise the system will behave in a way that is unespected

Reply:
Updated.